tests/robot-tests/tests/visual_testing/visually_check_permalinks.template.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Resource            ../libs/public-common.robot
+Resource            permalinks.robot
+
```

Comment:
Nice to see this template in use. Didn't realize RF allowed you to scaffold template test files

Reply:
Ahh, alas I'm afraid that this isn't a RF template, although I think they do exist in some form.  This is a template to be used by the generate_permalinks_test_suite.py script to generate a template containing multiple permalink URLs.  Would be nice to take some time to look at RF test templates though.